libcontainer/cgroups/systemd/common.go

```diff
@@ -355,15 +355,18 @@ func stopUnit(cm *dbusConnManager, unitName string) error {
 		return err
 	})
 	if err == nil {
+		timeout := time.NewTimer(30 * time.Second)
```

Comment:
Why did you switch away from `<-time.After(...)`? `<-time.After(30*time.Second)` is more idiomatic IMHO.

Reply:
Ah never mind, it's because you would need to wait for 30 seconds for the GC to free it. Fair point.